src/main/java/core/basesyntax/figures/Figure.java

```diff
@@ -0,0 +1,25 @@
+package core.basesyntax.figures;
+
+public abstract class Figure implements Drawable, Getarea {
+
+    protected String color;
+
+    public Figure(String color) {
+
+        this.color = color;
+    }
+
+    public String getColor() {
+
+        return color;
+    }
+
+    public void setColor() {
+
+        this.color = color;
+    }
+
+    public abstract double getArea();
+
+    public abstract void draw();
```

Comment:
no need to mention interface methods in abstract class if we do not plan to override them here
```suggestion
```